debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/EventQueue.java

```diff
@@ -180,7 +153,8 @@ private boolean shouldRotateQueueSegment() {
     private void rotateQueueSegment() {
         // close old file.
         try {
-            currentQueueSegment.close();
+            if (!currentQueueSegment.isClosed())
```

Comment:
Actually, we can just call 
```
 if isClosed() {
log("already closed")
return 
}
```
inside ` close()` . do you see any issue?

Reply:
Done